editor/app/(workbench)/[org]/[proj]/(console)/(campaign)/campaigns/[campaign]/design/page.tsx

```
@@ -46,14 +20,31 @@ export default function CampaignLayoutDesignerPage() {
+    return (
+      <div className="h-full flex items-center justify-center">
+        <span className="text-sm text-muted-foreground">
+          No template selected.
+        </span>
+      </div>
+    );
+  }
+
+  // NOTE: `template_id` here is the DB `template.id` (UUID), not the template key.
   const template =
     useWWWTemplate<TemplateData.West_Referrral__Duo_001>(template_id);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)